openedx/features/edly/api/v1/views/user_sites.py

```diff
@@ -0,0 +1,30 @@
+from rest_framework import viewsets
+from rest_framework.permissions import IsAuthenticated
+from rest_framework.response import Response
+
+from lms.djangoapps.mobile_api.decorators import mobile_view
+from openedx.features.edly.api.serializers import UserSiteSerializer
+
+
+@mobile_view()
+class UserSitesViewSet(viewsets.ViewSet):
```

Comment:
Please add detailed docstring for this viewset, like OpenedX did for its mobile API's `lms/djangoapps/mobile_api/users/views.py`

Reply:
Done